packages/server-core/src/projects/scene/scene.test.ts

```diff
@@ -23,190 +23,132 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
-        const { data } = await app
-          .service(sceneDataPath)
-          .get(null, { ...params, query: { ...params.query, projectName: defaultProjectName, metadataOnly: false } })
-        console.log('scene-data data', data)
-        assert.deepStrictEqual(parsedData, data.find((entry) => entry.name === defaultSceneName)!.scene)
-      })
+  after(async () => {
+    const foundProjects = (await app
+      .service(projectPath)
+      .find({ query: { name: projectName }, paginate: false })) as ProjectType[]
+    await app.service(projectPath).remove(foundProjects[0].id, { ...params })
+    await destroyEngine()
+  })
+
+  describe('"scene-data" service', () => {
+    it('should get the scene data', async () => {
+      const { data } = await app.service('scene-data').get(null, { query: { projectName, metadataOnly: false } })
+      assert.deepStrictEqual(parsedSceneData, data.find((scene) => scene.name === sceneName)!.scene)
     })
 
-    describe('find', () => {
-      it('should get all scenes for a project scenes', async function () {
-        const scenes = (await app.service(sceneDataPath).find({
-          ...params,
-          paginate: false
-        })) as SceneDataType[]
-        assert.deepStrictEqual(parsedData, scenes.find((entry) => entry.name === defaultSceneName)!.scene)
-        assert(scenes.length > 0)
-        scenes.forEach((scene) => {
-          assert(typeof scene.name === 'string')
-          assert(typeof scene.project === 'string')
-          assert(typeof scene.thumbnailUrl === 'string')
-          assert(typeof scene.scene === 'object')
-        })
+    it('should find the scene data', async () => {
+      const { data } = (await app
+        .service('scene-data')
+        .find({ query: { metadataOnly: false } })) as Paginated<SceneDataType>
+      assert.deepStrictEqual(parsedSceneData, data.find((entry) => entry.name === sceneName)!.scene)
+      assert(data.length > 0)
+      data.forEach((scene) => {
+        assert(typeof scene.name === 'string')
+        assert(typeof scene.project === 'string')
+        assert(typeof scene.thumbnailUrl === 'string')
+        assert(typeof scene.scene === 'object')
       })
+    })
 
-      it('should get all scenes for a project scenes with metadata only', async function () {
-        const scenes = (await app.service(sceneDataPath).find({
-          ...params,
-          query: { ...params.query, metadataOnly: true },
-          paginate: false
-        })) as SceneDataType[]
-        assert(scenes.length > 0)
-        scenes.forEach((scene) => {
-          assert(typeof scene.name === 'string')
-          assert(typeof scene.project === 'string')
-          assert(typeof scene.thumbnailUrl === 'string')
-          assert(typeof scene.scene === 'undefined')
-        })
+    it('should get all scenes for a project scenes with metadata only', async function () {
+      const { data } = (await app.service('scene-data').find({
```

Comment:
we should be using the path `sceneDataPath` for services

Reply:
implemented!